test/junit/src/test/scala/strawman/collection/ArrayOpsTest.scala

```diff
@@ -0,0 +1,20 @@
+package strawman.collection
+
+import org.junit.Assert.assertArrayEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@RunWith(classOf[JUnit4])
+class ArrayOpsTest {
+
+  @Test
+  def unzip(): Unit = {
+    val zipped = new ArrayOps(Array((1, 'a'), (2, 'b'), (3, 'c')))
+
+    val (a1: Array[Int], a2: Array[Char]) = zipped.unzip
```

Comment:
Does that also work if you don’t specify the expected types? Also, why not let the implicit conversion to `ArrayOps` do its job instead of explicitly converting to `ArrayOps`?

Reply:
>Does that also work if you don’t specify the expected types?

It does. I removed the types as `assertArrayEquals` is enough to make sure that what comes out of `unzip` is a tuple of arrays. 

>why not let the implicit conversion to ArrayOps do its job instead of explicitly converting to ArrayOps?

For some reason the implicit conversion from `Predef` is used when selecting `unzip`, even though they are unimported. Any idea?

Reply:
In the `junit` project `Predef` is not unimported by default so you have to unimport it manually.

Reply:
sorry, it turns out Intellij was tricking me. It showed the implicit conversions coming from `Predef` even though I had unimported them manually.